### ISSUE TITLE
KWord, compilation error after adding support for tvosSimulatorArm64()

### ISSUE THREAD

**Issue by marekkondracki** [mirego/trikot#192]
I have updated kword to the latest version(5.0.0), and tried to build my project. Unfortunately it failed, below stack trace.

```> Task :common:compileCommonMainKotlinMetadata FAILED
e: java.lang.IllegalStateException: e: Could not find "/Users/marek/path/to/project/common/build/kotlinTransformedMetadataLibraries/commonMain/com.mirego.trikot-kword-5.0.0-commonMain-TDK_lQ.klib" in [/Users/marek.pro.exe/Library/Application Support/kotlin/daemon]
        at org.jetbrains.kotlin.library.SingleFileResolveKt$resolveSingleFileKlib$1.fatal(SingleFileResolve.kt:21)
        at org.jetbrains.kotlin.library.KotlinLibrarySearchPathResolver.resolve(SearchPathResolver.kt:175)
        at org.jetbrains.kotlin.library.KotlinLibrarySearchPathResolver.resolve(SearchPathResolver.kt:181)
        at org.jetbrains.kotlin.library.CompilerSingleFileKlibResolveStrategy.resolve(SearchPathResolver.kt:303)
        at org.jetbrains.kotlin.library.SingleFileResolveKt.resolveSingleFileKlib(SingleFileResolve.kt:24)
        at org.jetbrains.kotlin.library.SingleFileResolveKt.resolveSingleFileKlib$default(SingleFileResolve.kt:15)
        at org.jetbrains.kotlin.cli.metadata.KlibMetadataDependencyContainer.<init>(K2MetadataKlibSerializer.kt:122)
        at org.jetbrains.kotlin.cli.metadata.K2MetadataKlibSerializer$serialize$analyzer$1.invoke(K2MetadataKlibSerializer.kt:54)
        at org.jetbrains.kotlin.cli.metadata.K2MetadataKlibSerializer$serialize$analyzer$1.invoke(K2MetadataKlibSerializer.kt:48)
        at org.jetbrains.kotlin.cli.metadata.CommonAnalysisKt.runCommonAnalysisForSerialization(CommonAnalysis.kt:42)
        at org.jetbrains.kotlin.cli.metadata.K2MetadataKlibSerializer.serialize(K2MetadataKlibSerializer.kt:53)
        at org.jetbrains.kotlin.cli.metadata.K2MetadataCompiler.doExecute(K2MetadataCompiler.kt:122)
        at org.jetbrains.kotlin.cli.metadata.K2MetadataCompiler.doExecute(K2MetadataCompiler.kt:40)
        at org.jetbrains.kotlin.cli.common.CLICompiler.execImpl(CLICompiler.kt:101)
        at org.jetbrains.kotlin.cli.common.CLICompiler.execImpl(CLICompiler.kt:47)
        at org.jetbrains.kotlin.cli.common.CLITool.exec(CLITool.kt:101)
        at org.jetbrains.kotlin.daemon.CompileServiceImpl.compile(CompileServiceImpl.kt:1645)
        at jdk.internal.reflect.GeneratedMethodAccessor99.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.base/java.lang.reflect.Method.invoke(Unknown Source)
        at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(Unknown Source)
        at java.rmi/sun.rmi.transport.Transport$1.run(Unknown Source)
        at java.rmi/sun.rmi.transport.Transport$1.run(Unknown Source)
        at java.base/java.security.AccessController.doPrivileged(Unknown Source)
        at java.rmi/sun.rmi.transport.Transport.serviceCall(Unknown Source)
        at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(Unknown Source)
        at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(Unknown Source)
        at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(Unknown Source)
        at java.base/java.security.AccessController.doPrivileged(Unknown Source)
        at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(Unknown Source)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.base/java.lang.Thread.run(Unknown Source)


FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':common:compileCommonMainKotlinMetadata'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Internal compiler error. See log for more details
```

**Comment by npresseault** [comment]
Would you happen to have a minimal reproducible project?

**Comment by marekkondracki** [comment]
It is minimum implementation copied from official example. It couldn't find "com.mirego.trikot-kword-5.0.0-commonMain-TDK_lQ.klib"

Revert back to version 4.4.0 works.

it also doesn't work with disabled tvosSimulatorArm64()

**Comment by npresseault** [comment]
We don't have any tvos sample at the moment. I'll add that to the todo list, might help diagnose the issue.

**Comment by marekkondracki** [comment]
@npresseault my project doesn't have also code for tvos, but we have it in the future plans. Now kword version 5.0.0 doesn't compile.

**Comment by npresseault** [comment]
I assume you mean your project doesn't compile with kword 5.0.0.
It does look like it might be a configuration problem on your side as we have samples with kword 5.0.0 and multiple projects using that. Maybe you aren't using the same version for all of trikot dependencies across your modules?

Like I said, if you have a minimum repro project, it would help, otherwise it seems a bug on your side.

**Comment by marekkondracki** [comment]
Problem solved, I have updated kotlin version to 1.9.0 and works!
